pkgs/development/compilers/binaryen/default.nix

```diff
@@ -4,13 +4,14 @@
 
 stdenv.mkDerivation rec {
   pname = "binaryen";
-  version = "118";
+  version = "119";
 
   src = fetchFromGitHub {
     owner = "WebAssembly";
     repo = "binaryen";
     rev = "version_${version}";
-    hash = "sha256-akMW3S2/qUyLK8F77EtnaXPDXvIMpkGfNB2jOD6hQho=";
+    hash = "sha256-JYXtN3CW4qm/nnjGRvv3GxQ0x9O9wHtNYQLqHIYTTOA=";
```

Comment:
Looks like this hash changed as a result of adding the submodule fetching.

Reply:
Latest force push addresses